package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "dashboardsMaps",
+  "version": "0.0.0",
```

Comment:
Shouldn't the two versions match? It's 0.0.0 here and 1.0.0 in the OpenSeach json

Reply:
yes, just got to know from @kavilla that these need to match the OpenSearch version. Will update accordingly.